src/Framework/test/SharedFxTests.cs

```diff
@@ -133,7 +133,7 @@ public void ItContainsValidDepsJson()
         [Fact]
         public void ItContainsVersionFile()
         {
-            var versionFile = Path.Combine(_sharedFxRoot, "Microsoft.AspNetCore.App.versions.txt");
+            var versionFile = Path.Combine(_sharedFxRoot, ".version");
```

Comment:
Tactical fix for now could be to just have a helix guard around the expected file I guess?  You can do a `string.IsNullOrEmpty(Environment.GetEnvironmentVariable("ASPNET_RUNTIME_PATH"))` check since this wouldn't be set locally to determine which file to look for

Reply:
Just pushed https://github.com/dotnet/aspnetcore/pull/21587/commits/582fe0deca0a5e480efe600fe4b86fbc5acd195b